src/osp/Resource/AssetImporter.cpp

```diff
@@ -243,6 +243,17 @@ DependRes<Mesh> osp::AssetImporter::compile_mesh(
     return pkg.add<Mesh>(meshData.name(), Magnum::MeshTools::compile(*meshData));
 }
 
+DependRes<Magnum::GL::Mesh> AssetImporter::compile_mesh(
+    const std::string& meshDataName, Package& package)
```

Comment:
Before anyone says anything, I know these should be `string_view`. I'll change it before merging.